scripts/plot_dispatch.py

```diff
@@ -65,6 +254,50 @@ def reduce_labels(ax, simple_labels_dict):
     return handles, labels
 
 
+def aggregate_by_region(bus_files):
```

Comment:
This function does more than its name suggests: It does not only aggregate, it also plots.

Reply:
Here is my proposition to solve this: commit https://github.com/rl-institut/oemof-B3/pull/242/commits/b7ae6b839bc4fb62efdda99e4d10bd8c2dc164ab.